std/random.d

```diff
@@ -1270,6 +1270,19 @@ alias Xorshift    = Xorshift128;                            /// ditto
     }
 }
 
+version(OpenBSD)
+{
+    version = ARC4_RANDOM;
+}
+else version(NetBSD)
+{
+    version = ARC4_RANDOM;
+}
```

Comment:
In recent version of OpenBSD/NetBSD, arc4random is supported by ChaCha20. On macOS it's ARC4.

OTH macOS's /dev/random uses Yarrow, so I guess that's a better choice?